client.js

```diff
@@ -156,34 +391,131 @@ bot.on('message', function (message) {
+                                    case '🔕':
+                                        // Ignore this seller's auctions for this watch
+                                        db.blockSeller(user.id, seller, null, watchId)
+                                        user.send(`Let's cut out the noise!  No longer notifying you about auctions from ${seller} with regard to this watch.\n  To block ${seller} on all present and future watches, use \`\`!add block: ${seller}`);
+                                        break;
+                                    case '♻': //extend watch
+                                        db.extendWatch(watchId)
+                                        user.send(`Good things come to those who wait.  I added another 7 days to your \`\`${formattedItem}\`\` watch.`);
+                                        break;
+                                    default:
+                                        break;
+                                }
+                            })
+                            collector.on('remove', (reaction, user) => {
+                                if (user.bot) return;
+                                switch (reaction.emoji.name) {
+                                    case '💤':
+                                        // unsnooze watch
+                                        db.unsnooze('watch', watch.id);
+                                        user.send(`Rise and grind.  No longer snoozing on your \`\`${item}\`\` watch on \`\`${server}\`\`.`).catch(console.error);
+                                        break;
+                                    case '❌':
+                                        // renew this watch
+                                        // db.endWatch(user.id, item, server);
+                                        user.send(`Got it! No longer watching auctions for ${item} on P1999 ${server} server.`);
+                                        break;
+                                    case '🔕':
+                                        // unblock the seller for this auction
+                                        db.unblockSeller(user.id, seller, null, watchId)
+                                        user.send(`Let's cut out the noise!  No longer notifying you about auctions from ${seller} with regard to this watch.\n  To block ${seller} on all present and future watches, use \`\`!add block: ${seller}`);
+                                        break;
+                                    default:
+                                        break;
+                                }
+                            })
+                        })
+                    })
+                    .catch(console.error);
+            })
+            .catch((err) => {
+                console.error(err)
+            });
     } else {
-        bot.users.cache.get(userID.toString()).send(msg)
+        bot.users.cache.get(user.toString()).send(msg)
```

Comment:
Wait, what? I'm finding it hard to process what this if/else was for... It looks like it's a TON of repeated code? What is it actually handling, the case where the user isn't able to be found in cache?
```
    if (bot.users.cache.get(user.toString()) === undefined) {
        bot.guilds.cache.get(GUILD).members.fetch(user.toString())
            .then((res)=>{
                res.send(msg)
                # More stuff happens
            }
    } else {
        bot.users.cache.get(user.toString()).send(msg)
        # Same stuff as above happens
    }
```

If that's it, this could be handled MUCH more cleanly -- just make the if/else grab the user, and once you have it, do all of this stuff to the result... Something like:
```
    if (bot.users.cache.get(user.toString()) === undefined) {
        bot.guilds.cache.get(GUILD).members.fetch(user.toString())
            .then((res)=>{
                actuallySendMessageFunction(res, msg);
            }
    } else {
        actuallySendMessageFunction(bot.users.cache.get(user.toString()), msg);
    }

function actuallySendMessageFunction(user_obj, message) {
    # All of that other stuff that was duplicated before
}
```

Reply:
You're right, you're right =)